grammar.ebnf

```diff
@@ -14,10 +14,13 @@ variable             ::= '$' identifier;
 keyword              ::= [^=|#{}\[\]()]+;
 builtin              ::= [A-Z_.?-]+;
 number               ::= [0-9]+ ('.' [0-9]+)?;
-member               ::= '*'? '[' keyword ']' __ pattern NL;
-member-list          ::= NL member+;
+member-base          ::= '[' keyword ']' __ pattern NL;
+default-member       ::= '*' member-base
+nondefault-member    ::= member-base
```

Comment:
I think this can be expressed by:

    member         ::= '[' keyword ']' __ pattern NL;
    default-member ::= '*' member